ultrasonic/src/main/kotlin/org/moire/ultrasonic/service/MediaPlayerService.kt

```diff
@@ -62,20 +69,25 @@ class MediaPlayerService : Service() {
     private val mediaPlayerLifecycleSupport by inject<MediaPlayerLifecycleSupport>()
 
     private var mediaSession: MediaSessionCompat? = null
-    private var mediaSessionToken: MediaSessionCompat.Token? = null
     private var isInForeground = false
     private var notificationBuilder: NotificationCompat.Builder? = null
 
+    val executorService: ExecutorService = Executors.newFixedThreadPool(4)
+
+    private val MEDIA_BROWSER_ROOT_ID = "_Ultrasonice_mb_root_"
+    private val MEDIA_BROWSER_ALBUM_LIST_ROOT = "_Ultrasonic_mb_album_list_root_"
+    private val MEDIA_BROWSER_ALBUM_PREFIX = "_Ultrasonic_mb_album_prefix_"
+    private val MEDIA_BROWSER_EXTRA_ENTRY_BYTES = "_Ultrasonic_mb_extra_entry_bytes_"
+
     private val repeatMode: RepeatMode
         get() = Util.getRepeatMode()
 
-    override fun onBind(intent: Intent): IBinder {
-        return binder
```

Comment:
This implementation actually doesn't work and caused me a lot of pain. The default provided by `MediaBrowserServiceCompat` works correctly. No need to override it